graphql/cxs-impl/src/main/java/org/apache/unomi/graphql/types/output/CDPInterest.java

```diff
@@ -19,37 +19,31 @@
 import graphql.annotations.annotationTypes.GraphQLField;
 import graphql.annotations.annotationTypes.GraphQLID;
 import graphql.annotations.annotationTypes.GraphQLName;
+import graphql.annotations.annotationTypes.GraphQLPrettify;
 
 @GraphQLName("CDP_Interest")
 public class CDPInterest {
 
-    @GraphQLID
-    @GraphQLField
     private String topic;
 
-    @GraphQLField
     private Double score;
 
-    public CDPInterest(@GraphQLID final String topic,
-                       final Double score) {
+    public CDPInterest(@GraphQLID final String topic, final Double score) {
         this.topic = topic;
         this.score = score;
     }
 
+    @GraphQLID
+    @GraphQLField
+    @GraphQLPrettify
```

Comment:
Use accessor without get instead of @GraphQLPrettify in entire class plz to be uniform across the app